### ISSUE TITLE
Initial new user creation fails in single-user (non-idp mode) with auth=tls

### ISSUE THREAD

**Issue by dmitrizagidulin** [solid/node-solid-server#381]
Having trouble creating a new user in non-idp mode. To reproduce, launch node-server in single user mode:

```
node bin/solid.js start -v --port 8443 --ssl-key=../localhost.key --ssl-cert=../localhost.cert --auth=tls --root=../accounts-new
```
Then load up `https://localhost:8443/` to start going through admin account creation. On the first page, enter email and hit submit. The log has the following:

```
  solid:settings Suffix Acl: .acl +0ms
  solid:settings Suffix Meta: .meta +2ms
  solid:settings Filesystem Root: ../accounts-new/ +1ms
  solid:settings Allow WebID registration (multi-user): true +0ms
  solid:settings Live-updates: true +0ms
  solid:settings Identity Provider: false +0ms
  solid:settings Default file browser app: https://linkeddata.github.io/warp/#/list/ +0ms
  solid:settings Default data browser app: true +1ms
  solid:settings Base URL (--mount): / +21ms
  solid:settings SSL Private Key path: ../localhost.key +0ms
  solid:settings SSL Certificate path: ../localhost.cert +0ms
Solid server (solid v3.0.1) running on https://localhost:8443/
Press <ctrl>+c to stop

  solid:idp Account on localhost is not available / +8s
  solid:authentication No client certificate found in the request. Did the user click on a cert? +3s
  solid:ACL Can an agent Append https://localhost:8443/accounts/new? +2ms
  solid:ACL Check if acl exist: https://localhost:8443/accounts/new.acl +2ms
  solid:ACL Error: HTTPError: Can't read file +1ms
  solid:ACL Check if acl exist: https://localhost:8443/accounts/.acl +1ms
  solid:ACL Error: HTTPError: Can't read file +0ms
  solid:ACL Check if acl exist: https://localhost:8443/.acl +0ms
  solid:ACL Error: HTTPError: Can't read file +0ms
  solid:ACL No ACL resource found - access not allowed +0ms
  solid:ACL Error: No Access Control Policy found +0ms
  solid:ACL Authentication required +0ms
  solid:server Error page because of Access to https://localhost:8443/accounts/new requires authorization +1ms
```

**Comment by dmitrizagidulin** [comment]
I just noticed the line `solid:settings Allow WebID registration (multi-user): true` even though I didn't pass `--idp` flag.

**Comment by dmitrizagidulin** [comment]
Looks like it's not just a single user mode issue, account creation fails in multi-user / `--idp` mode also:

```
solid:server Error page because of Access to https://dz.localhost:8443/accounts/new requires authorization
```

**Comment by dmitrizagidulin** [comment]
Fixed with PR #385. Closing.
